dashboard/app/controllers/api/v1/pd/application/facilitator_applications_controller.rb

```diff
@@ -12,12 +12,12 @@ def new_form
     protected
 
     def on_successful_create
-      @application.assign_default_workshop!
-      @application.assign_default_fit_workshop!
-      fit_workshop = @application.find_default_fit_workshop
-      @application.fit_workshop_id = fit_workshop.id if fit_workshop
+      # @application.assign_default_workshop!
+      # @application.assign_default_fit_workshop!
+      # fit_workshop = @application.find_default_fit_workshop
+      # @application.fit_workshop_id = fit_workshop.id if fit_workshop
```

Comment:
Should these lines be removed rather than commented?

Reply:
we can remove them - I have a checklist item to update this to the new way they want this to work this year.